scripts/sanity_chk/sanitylib.py

```diff
@@ -891,23 +891,31 @@ def handle(self):
                 #sometimes QEMU can't handle SIGTERM signal correctly
                 #in that case kill -9 QEMU process directly and leave
                 #sanitycheck judge testing result by console output
+                state, _ = self.get_state()
                 if os.path.exists(self.pid_fn):
                     qemu_pid = int(open(self.pid_fn).read())
                     try:
                         os.kill(qemu_pid, signal.SIGKILL)
                     except ProcessLookupError:
                         pass
```

Comment:
While you're here, can you add a log statement instead of the `pass`? Something like: "qemu process {qemu_pid} not found - just vanished?"

Reply:
Ok, thanks.